recipes/opencensus-context/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set opensensus_version = "0.7.10" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: opencensus-context
+  version: {{ version }}
+
+source:
+  url: https://github.com/census-instrumentation/opencensus-python/archive/v{{ opensensus_version }}.zip
+  sha256: 6b9a95efd0047e04d2d0b04efc0f38e4f2efdfa72746953cea7e586889f53e7a
+
+build:
+  noarch: python
+  number: 0
+  script: "cd context/opencensus-context && {{ PYTHON }} -m pip install . -vv"
```

Comment:
you probably don't need to cd

Reply:
Why not? That's where the source lives.

Reply:
Oh I thought that was why the build was failing